### DISCUSSION TITLE
Placeable Signs/Labels

### DISCUSSION THREAD

**Discussion by Boolean-Buckeye** [FakeFishGames/Barotrauma#12271] [Ideas and suggestions]
I think it would be very useful to have a placeable sign/label item that can be stuck on a wall and show a message. I know text display exists and it is useful for some purpose like when you want the display to change electronically. But this is a large and bulky display and requires a memory component at least to set the text, and only a few characters can be displayed before it starts scrolling. It would be nice to have a sign item that can display smaller/more characters and not scroll and not require (or be able to benefit from) wiring from components. It would just be a simple sticker that requires manual placement and not intended to be changed until deconstructed. This will help with labelling things such as buttons and switches or just leaving notes and warning or directions so that other/future crew members know what to do or not to do, or just give general information about the intended use of the submarine, or just leave some joke, really a lot of uses for something like this. 

To summarize, Reasons I do not think the text display is always sufficient or necessary:
It is too big, both the bulky frame and the large bold font (imagine using it to label what a switch added in game does, the proportions are way off)
It can not show very many characters at once
It requires wiring to wiring components to operate (at the very least a memory component)
The cost of the display and the memory component to have a bare minimum label seems too high (in my experience, tin is a hot commodity hard to come by in a campaign when you do any amount of wiring). Something like just a plastic should be the cost of placing a simple label that does not need to change dynamically or have fancy colors.
It aesthetically does not LOOK like just a simple label and seems out of place as one compared to signs that can be placed from the submarine editor (basically something like those are what I want but placeable in game)

**Comment by ghost** [comment]
To be honest, it's all in the game already, ready to be customized as necessary.
Take my SS13 themed [posters mod](https://steamcommunity.com/sharedfiles/filedetails/?id=3001707616). Use it as example. Make whatever stickers/posters you need.
